### PR TITLE
fix(schematics): fixed the schematics/action spec template

### LINKED ISSUE #2082: Failing action unit test without creators from NgRx schematic
When I use the action schematic to generate an action unit test with creators, I get a passing unit test. However, when I use the action schematic to generate an action unit test without creators, I get a failing unit test.

##Minimal reproduction of the bug/regression with instructions:

You get a failing spec when not using creators:
```
$ ng g @ngrx/schematics:action Add --project=counter --spec
$ npm test

ERROR in src/app/add.actions.spec.ts:1:10 - error TS2305: Module '"C:/Users/walrus95/proj/counter/src/app/add.actions"' has no exported member 'Add'.
1 import { Add } from './add.actions';
```

However, you get a passing spec when using creators:

```
$ ng g @ngrx/schematics:action Add --project=counter --spec --creators
$ npm test
TOTAL: 3 SUCCESS
```

## Expected behavior:

I would expect my newly created test to always be passing regardless of the use of creators, but at the very least it should be consistent.

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):

NgRx 8, Angular 8, Node 10.16.0

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No

Too busy swimming the arctic ocean. Splash, splash.

### PR DESCRIPTION
**Pull request by agugan**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [N/A] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

when you use the action schematic to generate an action unit test without creators, you get a failing unit test because of the wrong imports.

Closes #2082

## What is the new behavior?
Modified the spec template to properly import the actions and also uses the correct class template inside the test suites
## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { <%= classify(name) %> } from './<%= dasherize(name) %>.actions';
+import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
 
 describe('<%= classify(name) %>', () => {
   it('should create an instance', () => {
-    expect(new <%= classify(name) %>()).toBeTruthy();
+    expect(new from<%= classify(name)%>.Load<%= classify(name) %>s()).toBeTruthy();
```
Comment (timdeschryver) [inline modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]: Can you add a test for this?

Comment (agugan) [reply modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]: ok sure. The new test will check for the content generated by the action spec schematics?

Comment (timdeschryver) [reply modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]: Correct, for example https://github.com/ngrx/platform/blob/cdded0de149bea018b3760a791ed0a08b225ae34/modules/schematics/src/action/index.spec.ts#L102

Comment (agugan) [reply modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]: I have added the tests Tim. Can you check them ?

File: modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { <%= classify(name) %> } from './<%= dasherize(name) %>.actions';
+import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
```
Comment (brandonroberts) [inline modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]:
```suggestion
import * as <%= classify(name) %>Actions from './<%= dasherize(name) %>.actions';
```

File: modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template (hunk 1 above)
Comment (brandonroberts) [reply modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template]:
```suggestion
    expect(new<%= classify(name)%>Actions.Load<%= classify(name) %>s()).toBeTruthy();
```

File: modules/schematics/src/action/index.spec.ts [hunk 3]
```diff
@@ -114,6 +114,16 @@ describe('Action Schematic', () => {
...
     });
+
+    it('should create spec class with right imports', () => {
+      const options = { ...defaultOptions, spec: true };
+      const tree = schematicRunner.runSchematic('action', options, appTree);
+      const fileContent = tree.readContent(
+        `${projectPath}/src/app/foo.actions.spec.ts`
+      );
+
+      expect(fileContent).toMatch(/expect\(new fromFoo.LoadFoos\(\)\)/);
```
Comment (brandonroberts) [inline modules/schematics/src/action/index.spec.ts]:
```suggestion
      expect(fileContent).toMatch(/expect\(new FooActions.LoadFoos\(\)\)/);
```

**Comment by brandonroberts** [comment]
Thanks @agugan!

**Pull request merged**